docs/docs/behaviors/index.mdx

```diff
@@ -0,0 +1,72 @@
+| `&kp`         | [Key Press](key-press.md)                    | Send keycodes to the connected host when a key is pressed                                                                                                                                                                                         |
+| `&trans`      | [Transparent Key Press](misc.md#transparent) | Pass the keypress down to the next active layer in the stack for processing                                                                                                                                                                       |
+| `&none`       | [None Key Press](misc.md#none)               | Swallows and stops the keypress, no keycode will be sent nor will the keypress be passed down to the next active layer in the stack                                                                                                               |
+| `&mt`         | [Mod Tap](mod-tap.md)                        | Sends a different keypress, depending on whether it is held or tapped                                                                                                                                                                             |
+| `&gresc`      | [Mod Morph](mod-morph.md)                    | Sends a different keypress, depending on whether a specified modifier is being held during the keypress                                                                                                                                           |
+| `&kt`         | [Key Toggle](key-toggle.md)                  | Toggles the press of a key. If the key is not currently pressed, key toggle will press it, holding it until the key toggle is pressed again or the key is released in some other way. If the key is currently pressed, key toggle will release it |
+| `&sk`         | [Sticky Key](sticky-key.md)                  | A sticky key stays pressed until another key is pressed. It is often used for 'sticky shift'. By using a sticky shift, you don't have to hold the shift key to write a capital                                                                    |
+| `&caps_word`  | [Caps Word](caps-word.md)                    | The caps word behavior behaves similar to a caps lock, but will automatically deactivate when any key not in a continue list is pressed, or if the caps word key is pressed again                                                                 |
+| `&key_repeat` | [Key Repeat](key-repeat.md)                  | Send whatever keycode was last sent                                                                                                                                                                                                               |
```

Comment:
```suggestion
| `&sk`         | [Sticky Key](sticky-key.md)                  | Stays pressed until another key is pressed, then is released. It is often used for modifier keys like shift, which allows typing capital letters without holding it down                                                                                                                                    |
| `&caps_word`  | [Caps Word](caps-word.md)                    | Behaves similar to caps lock, but automatically deactivates when any key not in a continue list is pressed, or if the caps word key is pressed again                                                                 |
| `&key_repeat` | [Key Repeat](key-repeat.md)                  | Sends whatever keycode was last sent                                                                                                                                                                                                               |
```

Reply:
Great feedback! I will find some time in the coming days to work them in and push an update.